From: Mehdi Dogguy <mehdi@debian.org>
Date: Thu, 23 Aug 2018 21:48:58 +0200
Subject: Add a 'test' target

This patch is useful only for Debian: It helps dh_auto_test to detect
that there is a testsuite and run it accordingly. dh_auto_test checks
'test' and 'check' targets, but not 'tests'.
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index f24bf26..6b9619d 100644
--- a/Makefile
+++ b/Makefile
@@ -177,6 +177,7 @@ uninstall: opam.install
 	$(OPAMINSTALLER) -u $(OPAMINSTALLER_FLAGS) opam-installer.install
 
 .PHONY: tests
+test: tests
 tests: $(DUNE_DEP) src/client/no-git-version
 	@$(DUNE) runtest $(DUNE_PROFILE_ARG) --root . $(DUNE_ARGS) src/ tests/ --no-buffer; \
 	ret=$$?; \
